osgi.specs/docbook/core/058/service.resolver.xml

```diff
@@ -1415,6 +1416,144 @@ public List&lt;Capability&gt; findProviders( Requirement requirement) {
     </section>
   </section>
 
+  <section>
+    <title>Obtaining Resolvers Using SPI</title>
```

Comment:
To be consistent with other core sections that specify `META-INF/services` resources.

```suggestion
    <title>Java Service Provider Configuration Support for Resolvers</title>
```